File: Sakila_Net/VB_SakilaLib/Neg/NegInventory.vb
Imports  SakilaLib.Cl.Harrys.Sakila.DAO
Imports  SakilaLib.Cl.Harrys.Sakila.Model
Namespace Cl
    Namespace Harrys
        Namespace Sakila
            Namespace Neg

                Public Class ClNegInventory
                    Public Shared Function listar(nPagina as Integer ,cantidad as Integer) as DataTable
                        Return DAO.DAOInventory.SelectAll(nPagina, cantidad)	   
                    End Function
                    Public  Shared Function consultar( inventory as ClInventory) as Boolean
                        Return DAO.DAOInventory.SelectOne(inventory)     
                    End Function
                    Public  Shared Function crear( inventory as ClInventory) as Integer
                        Return DAO.DAOInventory.SqlInsert(inventory)     
                    End Function
                    Public  Shared Function eliminar( inventory as ClInventory) as Integer
                        Return DAO.DAOInventory.SqlDelete(inventory)     
                    End Function
                    Public  Shared Function actualizar( inventory as ClInventory) as Integer
                        Return DAO.DAOInventory.SqlUpdate(inventory)     
                    End Function
                End Class

            End Namespace  
        End Namespace  
    End Namespace  
End Namespace  

File: Sakila_Net/Sakila_Net/CS_SakilaLib/Neg/NegInventory.cs
using System.Data;
using Cl.Harrys.Sakila.DAO;
using Cl.Harrys.Sakila.Model;
namespace Cl.Harrys.Sakila.Neg
{
    public class ClNegInventory
    {
        public static DataTable listar(int nPagina ,int cantidad)
        {
            return DAO.DAOInventory.sqlSelectAll(nPagina, cantidad);
        }
        public  static bool consultar( ClInventory inventory) 
        {
            return DAO.DAOInventory.sqlSelect(inventory);
        }
        public  static int crear(ClInventory inventory)
        {
            return DAO.DAOInventory.sqlInsert(inventory);
        }
        public  static int eliminar(ClInventory inventory)
        {
            return DAO.DAOInventory.sqlDelete(inventory);
        }
        public  static int actualizar(ClInventory inventory)
        {
            return DAO.DAOInventory.sqlUpdate(inventory);
        }
    }
}
